addOns/soap/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- All rules have been tagged of interest to Penetration Testers, as well as adding tags associated with DEV or QA applicability.
```

Comment:
While not wrong, there's just one rule. The `DEV` should be dropped.

Reply:
Done